packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6040,6 +6040,20 @@ export default class Meeting extends StatelessWebexPlugin {
         EVENT_TRIGGERS.MEETING_MEDIA_REMOTE_STARTED,
         data
       );
+      if (data.type === 'share') {
+        // @ts-ignore
+        this.webex.internal.newMetrics.submitClientEvent({
+          name: 'client.media.render.start',
```

Comment:
BTW, there is also a "client.media.render.stop" event in the event dictionary. Is it OK that we're not sending it at all?

Reply:
Yes it is ok, Desktop barely sends this event. But you're right, there's no reason not to send in this case.